src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,36 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a Deadline task that is created upon user input.
+ */
+public class Deadline extends Task {
+    protected String by;
```

Comment:
Perhaps I would prefer a more informative naming for this variable such as "deadlineTime".